subprojects/packagefiles/lmdb/libraries/liblmdb/meson.build

```diff
@@ -0,0 +1,68 @@
+cc = meson.get_compiler('c')
+
+src = ['mdb.c', 'midl.c']
+
+c_args = ['-Wno-unused-parameter', '-Wbad-function-cast', '-Wuninitialized']
```

Comment:
Maybe instead of adding this manually to every target, you should just use `add_project_arguments`.

While you are at it, you can fix the MSVC error by using [`cc.get_supported_arguments()`](https://mesonbuild.com/Reference-manual_returned_compiler.html#compilerget_supported_arguments) to only pass those warning flags if the compiler supports them.

Reply:
Fixed